dls-controls/malcolmjs
# Feature/spline handling #277

## Description

Fixed spline handling so it is always horizontal for a source and sink port.
Fixed loop back so it is made up of lots of smaller segments to make the selection box smaller.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run the site and view a layout, move a block with a sink port on behind a block with a source port on. Check that the selection area for the link is only close to the line.

## Agile board tracking

connect to #277 
closes #277
